examples/3.x_api/pytorch/multimodal-modeling/quantization/auto_round/llama4/requirements.txt

```diff
@@ -0,0 +1,5 @@
+auto-tound==0.8.0
```

Comment:
auto_round @ git+https://github.com/intel/auto-round.git@v0.8.0rc, let's use this one before release.